### DISCUSSION TITLE
xz links have been removed from https://tukaani.org/xz/

### DISCUSSION THREAD

**Discussion by amorphobia** [ScoopInstaller/Main#4183] [General]
The latest available zip file is [version 5.2.8](https://tukaani.org/xz/xz-5.2.8-windows.zip)

The web page saved today: https://web.archive.org/web/20221201030639/https://tukaani.org/

**Comment by rashil2000** [comment]
See https://github.com/ScoopInstaller/Main/commit/804eda5b57876da9554657c484e39870c896ceda
